cmd/libs/go2idl/go-to-protobuf/protobuf/cmd.go

```diff
@@ -64,6 +64,7 @@ func New() *Generator {
 			`+k8s.io/kubernetes/pkg/api/resource`,
 			`+k8s.io/kubernetes/pkg/runtime`,
 			`k8s.io/kubernetes/pkg/api/unversioned`,
+			`+k8s.io/kubernetes/pkg/watch/versioned`,
```

Comment:
nit: can you please move this line one above (so that we have all "+" lines together)?